WeakAuras/GenericTrigger.lua

```diff
@@ -1399,6 +1400,10 @@ do
       update_frame:SetScript("OnUpdate", function()
         if not(WeakAuras.IsPaused()) then
           WeakAuras.ScanEvents("FRAME_UPDATE");
+          if (not rangeThrottle or GetTime() - rangeThrottle > 0.2) then
+            rangeThrottle = GetTime()
+            WeakAuras.ScanEvents("WA_SPELL_RANGECHECK")
+          end
```

Comment:
Is this an alright way to handle this throttled range check?

Reply:
Also this incidentally allows users to use this event as a throttled trigger. I know this was explicitly denied in the past but I would not know of a proper method to make this unavailable to the users.

Reply:
We usually factor it out into a component thatthen sends the events, but in your case that might be a bit overkill as the event sending is absolutely trivial.

There's some code for the existing range checking trigger that does send a event on a periodic timer already, maybe using that works. I haven't looked at it in detail.

I wouldn't worry about hiding that.

Reply:
Range Check is actually fired every frame. I recalled it being throttled as well but that isn't the case. The only other range checking is done via Conditions which uses its own system instead of events (https://github.com/WeakAuras/WeakAuras2/blob/master/WeakAuras/Conditions.lua#L688) and you have informed me in the past that Conditions don't receive ScanEvents.

Reply:
(I did also originally test with that event to verify it wasn't being received. I'm using the same event as the Condition.)

Reply:
Interesting, that is something that should be changed though.